### DISCUSSION TITLE
启动时出现错误

### DISCUSSION THREAD

**Discussion by GuihongWang** [SeaHOH/GotoX#216] [问答/咨询] [answered]
OSError: [WinError 10013] 以一种访问权限不允许的方式做了一个访问套接字的尝试
[log.txt](https://github.com/SeaHOH/GotoX/files/7963189/log.txt)

**Comment by SeaHOH** [comment, marked as answer]
如果你的系统是 Win10/11，那么这可能是因为当前配置的端口被划归到保留范围，换成数值更大的端口，或者使用系统命令查看排除保留端口都行。

有极小的可能是端口已经被其它应用占用，更改 GotoX 的配置即可。
